.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.2.3
+ruby-3.2.2
```

Comment:
Why is the ruby version being downgraded, are there issues with the higher version?

Reply:
I put it in the commit message, and I'll copy it here, too.

It looks like `ubuntu-latest` in GH actions is the latest patch of LTS
22, not LTS 24. I probably could add something to get updated packages
so 3.2.3 is available. I want to first check if the project has any
other problems before getting into the weeds

Reply:
This is part of the reason I have this PR still marked as draft